tests/test_pattoo_shared/test_daemon.py

```diff
@@ -22,10 +25,76 @@
     sys.exit(2)
 
 # Pattoo imports
-from pattoo_shared import daemon
+from pattoo_shared import files
+from pattoo_shared.daemon import Daemon, GracefulDaemon
+from pattoo_shared.agent import Agent
 from pattoo_shared.configuration import Config
 from tests.libraries.configuration import UnittestConfig
 
+# TEST CONSTANTS
+AGENT_NAME = 'parent'
+_config = Config()
+
+def handle_daemon(command):
```

Comment:
Why did you do this instead of just making the daemon_start_test_script a module and just run the required functions? I'd prefer if you use that approach versus this one if you don't see an overriding reason. I need feedback on this.

Reply:
The reason for doing this is because of how the Daemon functions are in terms of start/restart/daemonizing functions essentially fork and exit the initiall parent script, therefore if I call them directly in the unittest it will exit the unittest script through the sys.exit function within the daemonize function.